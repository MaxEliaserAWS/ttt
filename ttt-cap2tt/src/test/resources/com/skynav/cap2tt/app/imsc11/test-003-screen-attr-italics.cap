Lambda字幕V4	DF1+1	SCENE"和文標準"

1	10000000/10000100	斜体	＠正体
2	10000100/10000200	斜体	＠斜０
3	10000200/10000300	斜体	＠斜１
4	10000300/10000400	斜体	＠斜２
5	10000400/10000500	斜体	＠斜
6	10000500/10000600	斜体	＠斜３
7	10000600/10000700	斜体	＠斜４
8	10000700/10000800	斜体	＠斜５
