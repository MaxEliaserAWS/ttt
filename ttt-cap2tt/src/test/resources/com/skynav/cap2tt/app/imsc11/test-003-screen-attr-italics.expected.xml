<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ebutts="urn:ebu:tt:style" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" ebutts:multiRowAlign="start" tts:textAlign="center"/><style xml:id="s2" ebutts:multiRowAlign="start" tts:shear="6.345103%" tts:textAlign="center"/><style xml:id="s3" ebutts:multiRowAlign="start" tts:shear="11.33775%" tts:textAlign="center"/><style xml:id="s4" ebutts:multiRowAlign="start" tts:shear="16.78842%" tts:textAlign="center"/><style xml:id="s5" ebutts:multiRowAlign="start" tts:shear="21.99875%" tts:textAlign="center"/><style xml:id="s6" ebutts:multiRowAlign="start" tts:shear="27.97058%" tts:textAlign="center"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="10:00:00:00" end="10:00:01:00" style="s1">斜体</p><p begin="10:00:01:00" end="10:00:02:00" style="s1">斜体</p><p begin="10:00:02:00" end="10:00:03:00" style="s2">斜体</p><p begin="10:00:03:00" end="10:00:04:00" style="s3">斜体</p><p begin="10:00:04:00" end="10:00:05:00" style="s4">斜体</p><p begin="10:00:05:00" end="10:00:06:00" style="s4">斜体</p><p begin="10:00:06:00" end="10:00:07:00" style="s5">斜体</p><p begin="10:00:07:00" end="10:00:08:00" style="s6">斜体</p></div></body></tt>